src/graphiql-explain/useExplain.js

```diff
@@ -3,16 +3,87 @@ import { explainDataManager } from './ExplainDataManager'
 
 export const useExplain = () => {
   const [explain, setExplain] = useState(explainDataManager.getExplainData())
+  const [defaultExplain, setDefaultExplain] = useState(
+    explainDataManager.getExplainData()
+  )
+  const [pathOrder, setPathOrder] = useState(0)
+  const [timeOrder, setTimeOrder] = useState(0)
+
   useEffect(() => {
     const eventListener = explainDataManager.addEventListener(
       'updateExplainData',
       (e, value) => {
+        setDefaultExplain(() => e.target.explainData || [])
         setExplain(_ => e.target.explainData || [])
       }
     )
     return () => {
       explainDataManager.removeEventListener('updateExplainData', eventListener)
     }
   }, [])
-  return { explain }
+
+  const searchByField = fieldName => {
```

Comment:
add this in a useCalback. It's not currently strictly required but it's exported and if some component uses it will create an infinite loop.